src/internal/views/nunjucks/billing/macros/transactions-table.njk

```diff
@@ -36,6 +36,13 @@
   </td>
 {% endmacro %}
 
+{% macro minimumChargeDescription(transaction) %}
+  <td colspan="2" class="govuk-table__cell">
+    <h5 class="govuk-heading-s govuk-!-margin-bottom-0">Minimum Charge</h5>
+    <p class="govuk-body govuk-!-margin-bottom-0">Raised under Schedule 23 of the Environment Act 1995</p>
```

Comment:
Could this be:
```
 <p class="govuk-body govuk-!-margin-bottom-0">{{ transaction.description }}</p>
```
I thought I spotted the transaction description in the CM API response